app/utils/configuration/main.py

```diff
@@ -0,0 +1,76 @@
+import base64
+import configparser
+import os
+from pathlib import Path
+from typing import Union
+
+"""
+Notes:
+
+Ideally, this'd be a `class Configuration` so we can wrap ConfigParser()
+with some get/set methods and some robust config validation.
+
+Better yet, have a shared Configuration class/approach across our
+python tooling.
+"""
+
+# Note - might be better of as an env var? Wouldn't need a keyword or
+# default then
+DEFAULT_PATH = Path(Path(__file__).parent.parent.parent / 'configuration.ini')
+
+
+def get_config(logger, path: Union[str, Path] = DEFAULT_PATH):
+    """
+    Returns the basic parsed configuration.ini
+    Adds environmet variables.
+    """
+    if not isinstance(path, (str, Path)):
+        raise ValueError('get_config requires an argument of type Path or str')
+
+    if isinstance(path, str):
+        path = Path(path)
+    assert path.exists(
+    ), f'Specified config ini {path.absolute()} does not exist'
+
+    config = configparser.ConfigParser()
+    config.read(path)
+
+    """
+    Dev note:
+    Needs thought, but the neatest configuration solution might be to us the ENV
+    section in the configuration.ini for default values, so we check for an env
+    var override for each field - using the ini default in its absense.
+
+    There's a fine line between convenience and obfiscation here, if we do it we may
+    need logging out of sanitised variables, warnings per default use, something
+    like that.... think.
+    """
+
+    # Police current MVP approach
+    assert ["ENV"] not in config.sections(), (
```

Comment:
So if they do have "ENV", you'll complain that they must have the thing they have? I'm confused.

Reply:
this one is my bad from quick and dirty spike code 😄 , it was _aimed_ at enforce there is always an _empty_ section called `ENV` (I wanted to basically reserve it for dynamic population from env vars only - save future people tripping up on that bit of fuctionalty.).  

Still waking up but I think it needs to be `"ENV"` not `not ["ENV"]` to make sense.